### ISSUE TITLE
Customer MetaData Error

### ISSUE THREAD

**Issue by maplerichie** [woosignal/flutter-woocommerce-api#1]
API Response
```
 "meta_data": [
    {
      "id": 18,
      "key": "community-events-location",
      "value": {
        "ip": "173.245.211.0"
      }
    },
    {
      "id": 20,
      "key": "wc_last_active",
      "value": 1590796800
    }
  ],
```

Customer MetaData class
```
class MetaData {
  final int id;
  final String key;
  final String value;

  MetaData(this.id, this.key, this.value);

  MetaData.fromJson(Map<String, dynamic> json)
      : id = json['name'],
        key = json['email'],
        value = json['value'];

  Map<String, dynamic> toJson() => {'id': id, 'key': key, 'value': value};
}
```
Not compatiable when fromJson

**Comment by agordn52** [comment]
Resolved in v1.0.9 
Update the following dependancy in your pubspec.yaml

~~`woosignal: ^1.0.8`~~
`woosignal: ^1.0.9`

**Comment by maplerichie** [comment]
Alright, thanks
